common/keyboard.go

```diff
@@ -43,6 +43,7 @@ const (
 	ModifierKeyShift
 )
 
+//nolint:gochecknoglobals
```

Comment:
You can put `keyAToKeyZ` into a function and ask the function instead of this `nolint` directive. The Go compiler will most probably inline the function.

Besides, I'm not sure whether we need the map. You can simply create a function and return `true`/`false` depending on the pattern `Key[A-Z]`.